Core/Src/torque_command/torque_manager.c

```diff
@@ -61,16 +57,32 @@ void torque_manager(void* argument) {
                 osDelayUntil(tick);
 
                 break;
```

Comment:
Isso era do switch antigo. Como não tem nenhum outro escopo que aceite `break` aqui além do do `for`, isso vai fazer a tarefa sair da função. Isso faz o RTOS parar de funcionar. De novo, eu espero que você volte isso para um esquema de `switch`, mas cuidado com esse tipo de problema porque é extremamente difícil de debugar (falo por experiência própria rs)